radio/src/dataconstants.h

```diff
@@ -670,6 +670,10 @@ enum MixSources {
   MIXSRC_CYC2,                         LUA_EXPORT("cyc2", "Cyclic 2")
   MIXSRC_CYC3,                         LUA_EXPORT("cyc3", "Cyclic 3")
 
+#if defined(DEBUG_LATENCY_MIXER_RF)
+  MIXSRC_DEBUG_SINUS,
```

Comment:
Why not at the end of the list?

Reply:
I had troubles to have it included in the sources accepted by the mixer screen.

Reply:
There should be only one place, isSourceAvailable()

Reply:
Should, but it does not help, as this only works for the correct categories. So, placing this at the end requires including more categories of inputs, or even “creating” one.